code/game/machinery/doors/airlock.dm

```diff
@@ -1604,6 +1605,30 @@
 	else if(istype(note, /obj/item/photo))
 		return "photo"
 
+/obj/machinery/door/airlock/AltClick(mob/living/user)
+	if(!istype(user))
+		to_chat(user, "<span class='info'>Nice try, ghosts.</span>")
+		return
+
+	if (!user.canUseTopic(src))
+		to_chat(user, "<span class='info'>You can't do this right now!</span>")
+		return
+
+	if(stat & (NOPOWER|BROKEN) || emagged)
+		to_chat(user, "<span class='info'>The door isn't working!</span>")
+		return
+
+	if(request_cooldown > world.time)
+		to_chat(user, "<span class='info'>The airlocks spam filter is blocking your request. Please wait at least 10 seconds between requests.</span>")
```

Comment:
airlock's

Reply:
Hijacking your comment. Should it complain, subtly or otherwise if the AI wire is cut? Or just let you think the AI can see it until you sum up the courage to _;AI stop ignore me!_